packages/animations/example/lib/main.dart

```diff
@@ -74,7 +74,7 @@ class _TransitionsHomePageState extends State<_TransitionsHomePage> {
                 ),
                 _TransitionListTile(
                   title: 'Container Transform',
-                  subtitle: 'The container acts as a persistent '
+                  subtitle: 'OpenContainer'
                       'element as its dimensions, position, and '
```

Comment:
Also, remove the rest of the text that comes after 'OpenContainer' :)

Reply:
lol, I blame this one on github :) 

Done.